packages/rich-text/src/create.js

```diff
@@ -178,6 +178,19 @@ export class RichTextData {
 	}
 }
 
+for ( const name of Object.getOwnPropertyNames( String.prototype ) ) {
```

Comment:
Do you think converting all methods like that is good? I mean, there's a huge chance that most these methods will create sync issues between the string and the object no? Maybe we should convert case by case.

Reply:
What do you mean with sync issues? Sure, I can hardcode the list here instead if you prefer.